sdk/scheduler/src/main/java/com/mesosphere/sdk/state/SchemaVersionStore.java

```diff
@@ -18,6 +17,16 @@
  */
 public class SchemaVersionStore {
 
+    /**
```

Comment:
Actually, how about defining a `SchemaVersion` enum, and then use that enum instead of passing around raw ints directly?

It could look something like this, where the enum type could even include associated utility methods:

```
public enum SchemaVersion {
  SINGLE_SERVICE,
  MULTI_SERVICE,
  UNKNOWN;

  public static SchemaVersion parseInt(int rawVersion) {
    switch (rawVersion) {
      case 1:
        return SINGLE_SERVICE;
      case 2:
        return MULTI_SERVICE;
      default:
        return UNKNOWN;
    }
  }

  public int toInt() {
    switch (this) {
      case SINGLE_SERVICE:
        return 1;
      case MULTI_SERVICE:
        return 2;
      default:
        throw new IllegalArgumentException(String.format("Unable to convert %s to int", this));
    }
  }
};
```

Then the SchemaVersionStore would use the enum type in its interfaces:
```
public void check(SchemaVersion expectedVersion) {
  ...
}

public SchemaVersion getOrSetVersion(SchemaVersion expectedVersion) {
  ...
}
```

Also, the `getSupported*()` calls wouldn't be needed -- upstream could compare `SchemaVersion` enums directly